src/api/index.js

```diff
@@ -58,7 +58,16 @@ api.post('/profile', async (req, res, next) => {
     if (!dbUser) {
       throw new errors.NotFound();
     }
-    dbUser.update(req.body);
+    // basic channel name sanitization
+    let channel = req.body.channel;
+    if (!/^[a-zA-Z0-9\-_]{1,64}$/.test(channel)){
+      channel = null;
+    }
+    dbUser.update({
```

Comment:
While we're here, can we make this?:

``` js
await dbUser.update({
```

It should've been like that from the beginning, since this is an `async` function.